pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -166,47 +166,50 @@ class Build {
     def sign(VersionInfo versionInfo) {
         // Sign and archive jobs if needed
         // TODO: This version info check needs to be updated when the notarization fix gets applied to other versions.
-        if (buildConfig.TARGET_OS == "windows" || (buildConfig.TARGET_OS == "mac" && versionInfo.major == 8 && buildConfig.VARIANT != "openj9") || (buildConfig.TARGET_OS == "mac" && versionInfo.major == 13)) {
-            context.node('master') {
-                context.stage("sign") {
-                    def filter = ""
-                    def certificate = ""
-
-                    def nodeFilter = "${buildConfig.TARGET_OS}"
-
-                    if (buildConfig.TARGET_OS == "windows") {
-                        filter = "**/OpenJDK*_windows_*.zip"
-                        certificate = "C:\\Users\\jenkins\\windows.p12"
-                        nodeFilter = "${nodeFilter}&&build"
-
-                    } else if (buildConfig.TARGET_OS == "mac") {
-                        filter = "**/OpenJDK*_mac_*.tar.gz"
-                        certificate = "\"Developer ID Application: London Jamocha Community CIC\""
+        if (
```

Comment:
This seems very specific so only for AdoptOpenJDK 8 (OpenJ9) on Mac and Adoptopenjdk 13 (Hotspot) on Mac or all Windows?  This seems very odd

Reply:
I haven't changed that line other than adding some newlines to make it more readable. If you hide the whitespace changes on the diff, it might be a little more readable

Reply:
Agreed, I still want to figure out if this is still the right thing to do though :-) - @gdams can you comment?  The version == 13 is the most concerning part...

Reply:
that is correct, its to do with the notarization backports